apps/avatax/scripts/run-webhooks-migration.ts

```diff
@@ -40,60 +40,58 @@ const runMigrations = async () => {
 
-        loggerContext.set("dryRun", dryRun.toString());
-        loggerContext.set(ObservabilityAttributes.SALEOR_API_URL, saleorApiUrl);
+      loggerContext.set("dryRun", dryRun.toString());
+      loggerContext.set(ObservabilityAttributes.SALEOR_API_URL, saleorApiUrl);
 
-        logger.info(`Migrating webhooks for ${saleorApiUrl}`);
+      logger.info(`Migrating webhooks for ${saleorApiUrl}`);
 
-        const client = createInstrumentedGraphqlClient({
-          saleorApiUrl: saleorApiUrl,
-          token: token,
-        });
-
-        const runner = new WebhookMigrationRunner({
-          dryRun,
-          logger,
-          client,
-          getManifests: async ({ appDetails, instanceDetails }) => {
-            if (instanceDetails.version) {
-              loggerContext.set(
-                ObservabilityAttributes.SALEOR_VERSION,
-                instanceDetails.version?.toFixed(2),
-              );
-            }
+      const client = createInstrumentedGraphqlClient({
+        saleorApiUrl: saleorApiUrl,
+        token: token,
+      });
 
-            const webhooks = appDetails.webhooks;
+      const runner = new WebhookMigrationRunner({
+        dryRun,
+        logger,
+        client,
+        getManifests: async ({ appDetails, instanceDetails }) => {
+          if (instanceDetails.version) {
+            loggerContext.set(
+              ObservabilityAttributes.SALEOR_VERSION,
+              instanceDetails.version?.toFixed(2),
+            );
+          }
 
-            if (!webhooks?.length) {
-              logger.warn("The environment does not have any webhooks, skipping");
-              return [];
-            }
+          const webhooks = appDetails.webhooks;
 
-            const enabled = webhooks.some((w) => w.isActive);
+          if (!webhooks?.length) {
+            logger.warn("The environment does not have any webhooks, skipping");
+            return [];
+          }
 
-            const targetUrl = appDetails.appUrl;
+          const enabled = webhooks.some((w) => w.isActive);
 
-            if (!targetUrl?.length) {
-              logger.error("App has no defined appUrl, skipping");
-              return [];
-            }
+          const targetUrl = appDetails.appUrl;
 
-            const baseUrl = new URL(targetUrl).origin;
+          if (!targetUrl?.length) {
+            logger.error("App has no defined appUrl, skipping");
+            return [];
+          }
 
-            // All webhooks in this application are turned on or off. If any of them is enabled, we enable all of them.
-            return appWebhooks.map((w) => ({ ...w.getWebhookManifest(baseUrl), enabled }));
-          },
-        });
+          const baseUrl = new URL(targetUrl).origin;
 
-        await runner.migrate();
+          // All webhooks in this application are turned on or off. If any of them is enabled, we enable all of them.
+          return appWebhooks.map((w) => ({ ...w.getWebhookManifest(baseUrl), enabled }));
```

Comment:
if you touch this file, you can also:

```suggestion
          return appWebhooks.map((w) => ({ ...w.getWebhookManifest(baseUrl), isActive: enabled }));
```